includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -184,6 +184,105 @@ private function get_learner_courses_page_template() {
+
+<!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column {"width":"66.66%"} -->
+<div class="wp-block-column" style="flex-basis:66.66%"><!-- wp:sensei-lms/course-categories {"textAlign":"left","options":{},"style":{"spacing":{"padding":{"bottom":"var:preset|spacing|20"}}}} -->
+<div style="padding-bottom:var(--wp--preset--spacing--20)" class="wp-block-sensei-lms-course-categories has-text-align-left"></div>
+<!-- /wp:sensei-lms/course-categories -->
+
+<!-- wp:post-title {"textAlign":"left","isLink":true,"style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}}} /-->
+
+<!-- wp:post-author {"textAlign":"left"} /-->
+
+<!-- wp:post-excerpt {"textAlign":"left"} /-->
+
+<!-- wp:sensei-lms/course-progress {"defaultBarColor":"foreground","className":"has-text-align-left"} /--></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"33.33%","style":{"spacing":{"padding":{"top":"0"}}}} -->
+<div class="wp-block-column" style="padding-top:0;flex-basis:33.33%"><!-- wp:sensei-lms/course-actions -->
+<!-- wp:sensei-lms/button-take-course {"align":"right"} -->
+<div class="wp-block-sensei-lms-button-take-course is-style-default wp-block-sensei-button wp-block-button has-text-align-right"><button class="wp-block-button__link">' . __( 'Start Course', 'sensei-lms' ) . '</button></div>
+<!-- /wp:sensei-lms/button-take-course -->
+
+<!-- wp:sensei-lms/button-continue-course {"align":"right"} -->
+<div class="wp-block-sensei-lms-button-continue-course is-style-default wp-block-sensei-button wp-block-button has-text-align-right"><a class="wp-block-button__link">' . __( 'Continue', 'sensei-lms' ) . '</a></div>
+<!-- /wp:sensei-lms/button-continue-course -->
+
+<!-- wp:sensei-lms/button-view-results {"align":"right","className":"is-style-default"} -->
+<div class="wp-block-sensei-lms-button-view-results is-style-default wp-block-sensei-button wp-block-button has-text-align-right"><a class="wp-block-button__link">' . __( 'Visit Results', 'sensei-lms' ) . '</a></div>
+<!-- /wp:sensei-lms/button-view-results -->
+<!-- /wp:sensei-lms/course-actions --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns --></div>
+<!-- /wp:group -->
+<!-- /wp:post-template -->
+
+<!-- wp:query-pagination {"paginationArrow":"arrow","align":"center","layout":{"type":"flex","justifyContent":"space-between"}} -->
+<!-- wp:query-pagination-previous {"fontSize":"small"} /-->
+
+<!-- wp:query-pagination-numbers /-->
+
+<!-- wp:query-pagination-next {"fontSize":"small"} /-->
+<!-- /wp:query-pagination --></div>',
+						],
+						'attrs'        => [
+							'queryId'       => 0,
+							'query'         => [
+								'postType' => 'course',
+								'perPage'  => 3,
```

Comment:
I don't quite get why this is set to 3, but it seems to work regardless (i.e. uses the _Blog pages show at most value_). Should we set it to 10 just for consistency's sake with My Courses? Do we even need to set it?

Reply:
Yeah the "perPage" value is ignored when query is inherited. I've removed all the values it can function without here https://github.com/Automattic/sensei/pull/6354/commits/6b980060b012015e952515975481922557070a78. Thanks!